src/studio/src/designer/frontend/packages/schema-editor/src/utils.ts

```diff
@@ -152,6 +190,9 @@ export function buildUISchema(schema: any, rootPath: string, includeDisplayName:
         type,
         items,
         enum: enums,
+        oneOf: oneOf?.map((child: object, index: number) => mapGroupItemToUiSchemaItem(child, index, 'oneOf', path)),
+        allOf: allOf?.map((child: object, index: number) => mapGroupItemToUiSchemaItem(child, index, 'allOf', path)),
+        anyOf: anyOf?.map((child: object, index: number) => mapGroupItemToUiSchemaItem(child, index, 'anyOf', path)),
```

Comment:
Maybe these could be replaced with something like this:
```js
...getCombo('oneOf'),
...getCombo('allOf'),
...getCombo('anyOf'),
```
and the have a function on line 184
```js
const getCombo(coboType) {
  return restrictions[comboType] && {
    [coboType]: restrictions[comboType].map((c, i) => mapGroupItemToUiSchemaItem(c, i, comboType, path))
 }
}
```